app/assets/stylesheets/common/base/menu-panel.scss

```diff
@@ -108,6 +108,15 @@
     }
   }
 
+  ul.category-links {
```

Comment:
do we need `ul` and `li` here ? it's adding specificity which might make styling harder at some point